internal/kafka/command_consumer_group_lag.go

```diff
@@ -0,0 +1,39 @@
+package kafka
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/confluentinc/cli/v3/pkg/config"
+)
+
+type lagOut struct {
+	ClusterId       string `human:"Cluster" serialized:"cluster"`
```

Comment:
```suggestion
	Cluster         string `human:"Cluster" serialized:"cluster"`
```

Reply:
I actually just added `_id` to the serialized field instead so that these fields match the `group` output fields.